src/lookup-table/services/LookupTable.service.ts

```diff
@@ -24,15 +25,15 @@ const LookupTableService = async (props: LookupTableServiceProps) => {
   if (props.tableId) {
     tableId = props.tableId
     structure = await lookupTableModel.getStructureById(tableId)
-  } else if (props.tableNameLabel) {
-    tableNameLabel = props.tableNameLabel
+  } else if (props.name) {
+    name = props.name
   }
 
   // Exported Methods
   const getAllRowsForTable = async () => await lookupTableModel.getAllRowsForTable(structure)
 
   const createTable = async () => {
-    tableName = toSnakeCase(tableNameLabel)
+    tableName = toSnakeCase(singular(name))
```

Comment:
Singular table names enforced in back-end. It really does mess things up (because of Postgraphile inflections) if we give a table a plural name, so it's pretty important to do this.